tuktuk-sdk/src/tuktuk.rs

```diff
@@ -32,6 +32,14 @@ pub fn task_queue_name_mapping_key(config_key: &Pubkey, name: &str) -> Pubkey {
     .0
 }
 
+pub fn custom_signing_key(task_queue: &Pubkey, signer_seeds: &[&[u8]]) -> Pubkey {
```

Comment:
```suggestion
pub fn custom_signer_key(task_queue: &Pubkey, signer_seeds: &[&[u8]]) -> Pubkey {
```

To be consistent with typescript land

Reply:
fixed in [dd4f50b](https://github.com/helium/tuktuk/pull/60/commits/dd4f50bd57949d6b419c1538e703a5aaaae628b2)